online-shop/src/app/cart/cart-menu/cart-menu.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-cart-menu',
+  templateUrl: './cart-menu.component.html'
+})
+export class CartMenuComponent {
+  @Output() checkout = new EventEmitter();
+
+  constructor() { }
+
+  doCheckout() {
+    this.checkout.emit();
```

Comment:
Just a small hint: you can directly do `checkout.emit()` in the html template instead of having a dedicated method.